testsuite/k8s-conformance-test-suite.sh

```diff
@@ -78,12 +152,21 @@ while IFS= read -r arc_platform_version || [ -n "$arc_platform_version" ]; do
     containerString="conformance-results-major-${version[0]}-minor-${version[1]}-patch-${version[2]}"
     IFS=$' \t\n'
 
-    az storage container create -n $containerString --account-name $AZ_STORAGE_ACCOUNT --sas-token $AZ_STORAGE_ACCOUNT_SAS
-    az storage blob upload --file conformance-results-$arc_platform_version.tar.gz --name conformance-results-$OFFERING_NAME.tar.gz --container-name $containerString --account-name $AZ_STORAGE_ACCOUNT --sas-token $AZ_STORAGE_ACCOUNT_SAS
+    az_storage_create_result=$(az storage container create -n $containerString --account-name $AZ_STORAGE_ACCOUNT --sas-token $AZ_STORAGE_ACCOUNT_SAS)
+    az_storage_upload_result=$(az storage blob upload --file conformance-results-$arc_platform_version.tar.gz --name conformance-results-$OFFERING_NAME.tar.gz --container-name $containerString --account-name $AZ_STORAGE_ACCOUNT --sas-token $AZ_STORAGE_ACCOUNT_SAS)
+
+    echo "az_storage_create_result : $az_storage_create_result"
+    echo "az_storage_upload_result : $az_storage_upload_result"
 
     echo "Cleaning the cluster.."
     sonobuoy delete --wait
 
+    if [[ $az_storage_create_result==*"ERROR"* || $az_storage_upload_result==*"ERROR"* ]]
```

Comment:
This step needs to go before running 'sonobuoy delete' otherwise the results will anyways be lost